package.xml

```diff
@@ -56,64 +56,50 @@
         <api>${version}</api>
     </version>
     <stability>
-        <release>stable</release>
+        <release>beta</release>
         <api>stable</api>
     </stability>
     <license uri="https://github.com/DataDog/dd-trace-php/blob/master/LICENSE">BSD 3-Clause</license>
     <notes>
         <![CDATA[
-## Tracer
-
-Windows x64 support is officially GA (generally available).
+This beta release contains all breaking changes planned for dd-trace-php 1.0. An upgrade guide can be found at https://github.com/DataDog/dd-trace-php/blob/1.0.0beta1/UPGRADE-1.0.md.
```

Comment:
Question: Is 1.0.0beta1 the right blob to use? Will it get created later on?

Reply:
Yes.